app/assets/javascripts/effective_datatables/initialize.js.coffee

```diff
@@ -49,7 +48,7 @@ initializeDataTables = (target) ->
       deferRender: true
       displayStart: datatable.data('display-start')
       iDisplayLength: datatable.data('display-length')
-      language: { 'lengthMenu': '&nbsp;with _MENU_ per page'}
+      language: { url: "/datatables/i18n/" + datatable.data('locale') }
```

Comment:
there's gotta be some way we can detect this and set the language without an extra page request?